openldap/README.md

```diff
@@ -0,0 +1,97 @@
+# OpenLDAP Sample App
```

Comment:
I think it would be helpful to mention supported platforms. (Linux, Windows) (x86, arm64)

We have previously had issues with apps on M1

Reply:
Updated. I am currently working on a M2, so I assume M2+ is good.